src/parenttext_pipeline/configs.py

```diff
@@ -0,0 +1,198 @@
+from dataclasses import dataclass, field
+
+@dataclass(kw_only=True)
+class StepConfig:
+    # Identifier (name) of the step
+    id: str
+    # Type of the step, should be one of STEP_CONFIGS.keys()
+    # Make this an enum maybe
+    type: str
+    # A list of input data sources used by this step
+    sources: list = field(default_factory=list)
+
+
+@dataclass(kw_only=True)
+class CreateFlowsStepConfig(StepConfig):
+    # Name of the Python module containing data models describing the data sheets
+    models_module: str = None
```

Comment:
Do we possibly want to store the models with the sources rather than the steps, so that if necessary, we can do sanity checks already when pulling? Or even convert to nested representations? We then still need meta-information associating models to sheets, which is not straightfoward. In RPFT this is done via the content index: certain types are known, and data sheets specifically have their model specified in a column.

Reply:
We should move model definitions from Python into the spreadsheets. I think this is the only option really worth considering.